submissions/kuslosolodka/html-css-popup/index.html

```diff
@@ -0,0 +1,199 @@
+          >
+        </li>
+        <li>
+          <a href="#" class="menu-icon"
+            ><img
+              class="icon-photo"
+              src="assets/icons/videos.png"
+              alt="videos icon"
+            />Videos</a
+          >
+        </li>
+        <li>
+          <a href="#" class="menu-icon"
+            ><img
+              class="icon-photo"
+              src="assets/icons/search.png"
+              alt="search icon"
+            />Search</a
+          >
+        </li>
+        <li>
+          <a href="#" class="menu-icon"
+            ><img
+              class="icon-photo"
+              src="assets/icons/photos.png"
+              alt="two people: yellow and blue icon"
+            />Photo</a
+          >
+        </li>
+        <li>
+          <a href="#" class="menu-icon"
+            ><img
+              class="icon-photo"
+              src="assets/icons/apps.png"
+              alt="circle with four colors: blue, green, yellow, red icon"
+            />Apps</a
+          >
+        </li>
+        <li>
+          <a href="#" class="menu-icon"
+            ><img
+              class="icon-photo"
+              src="assets/icons/documents.png"
+              alt="documents icon"
+            />Document</a
+          >
+        </li>
+      </ul>
+      <input type="checkbox" id="load-more-btn" />
+      <label class="load-more" for="load-more-btn">
+        <p class="btn-text">More</p>
+      </label>
+      <ul class="hidden-icons">
+        <li>
+          <a href="#" class="menu-icon"
+            ><img
+              class="icon-photo"
+              src="assets/icons/chat.png"
+              alt="chat icon"
+            />Chat</a
+          >
+        </li>
+        <li>
+          <a href="#" class="menu-icon"
+            ><img
+              class="icon-photo"
+              src="assets/icons/classroom.png"
+              alt="classroom icon"
+            />Classroom</a
+          >
+        </li>
+        <li>
+          <a href="#" class="menu-icon"
+            ><img
+              class="icon-photo"
+              src="assets/icons/collection.png"
+              alt="collection icon"
+            />Collection</a
+          >
+        </li>
+        <li>
+          <a href="#" class="menu-icon"
+            ><img
+              class="icon-photo"
+              src="assets/icons/mars.png"
+              alt="mars icon"
+            />Mars</a
+          >
+        </li>
+        <li>
+          <a href="#" class="menu-icon"
+            ><img
+              class="icon-photo"
+              src="assets/icons/blogs.png"
+              alt="blogs icon"
+            />Blogs</a
+          >
+        </li>
+        <li>
+          <a href="#" class="menu-icon"
+            ><img
+              class="icon-photo"
+              src="assets/icons/promo.png"
+              alt="promo icon"
+            />Promos</a
+          >
+        </li>
+        <p class="footer">Even more from Kottans</p>
```

Comment:
Logically it must be a link, and fix elements width:
![image](https://user-images.githubusercontent.com/16196199/185970257-67f06857-571d-40af-b20a-03d5e1c0ef92.png)